RUST_VERSION

```diff
@@ -0,0 +1,2 @@
+# This file is used for specifying Rust version for Jenkins CI
+nightly-2018-01-12
```

Comment:
Can circle ci reuse this?

Reply:
I think that's a great idea. Might be good to have the rustfmt version as well somehow.

Reply:
If rustfmt is installed via `rustup component`, then version is unnecessary anymore.